libs/moloch-v3-macro-ui/src/components/MemberDisplay/MemberDisplay.tsx

```diff
@@ -0,0 +1,64 @@
+import { useCallback, useEffect, useState } from 'react';
+import { AccountProfile } from '@daohaus/utils';
+import { Keychain } from '@daohaus/keychain-utils';
+
+import {
+  MemberCard,
+  MemberCardCopyAddress,
+  MemberCardExplorerLink,
+  MemberCardItem,
+} from '@daohaus/ui';
+import { MemberContainer, StyledRouterLink } from './MemberDisplay.styles';
+import { useProfile } from '@daohaus/moloch-v3-hooks';
+
+type MemberProfileProps = {
+  memberAddress: string;
+  daoChain: keyof Keychain;
+  daoId?: string;
+  includeLinks?: boolean;
+};
+
+export const MemberDisplay = ({
+  memberAddress,
+  daoChain,
+  daoId,
+  includeLinks = false,
+}: MemberProfileProps) => {
+  const { profile } = useProfile({ address: memberAddress });
+
+  const customProfileURI =
+    daoId && daoId && `/molochv3/${daoChain}/${daoId}/members/${memberAddress}`;
+
+  return (
+    <MemberContainer>
+      <MemberCard
+        fullWidth={true}
+        variant="ghost"
+        profile={
+          profile || {
+            address: memberAddress,
+          }
+        }
+      >
+        <MemberCardItem asChild>
+          {includeLinks && (
+            <StyledRouterLink to={customProfileURI ? customProfileURI : '/'}>
+              View Profile
+            </StyledRouterLink>
+          )}
+        </MemberCardItem>
```

Comment:
Bad things could happen if try and render the MemberCardItem asChild if there is no child. Just move the check up
```suggestion
        {includeLinks && (
          <MemberCardItem asChild>
            <StyledRouterLink to={customProfileURI ? customProfileURI : '/'}>
              View Profile
            </StyledRouterLink>
          </MemberCardItem>
        )}
```